e2e/cypress/integration/dcat.cy.js

```diff
@@ -0,0 +1,26 @@
+describe('DCAT Extension', () => {
+    const orgId = "org-" + cy.helpers.randomSlug();
+    const packageId = "dataset-" + orgId;
+
+    before(() => {
+      cy.login();
+      cy.create_token();
+      cy.create_organization(orgId);
+      cy.create_dataset({
+        "name": packageId,
+        "owner_org": orgId,
+      });
+    });
+
+    after(() => {
+      cy.delete_dataset(packageId);
+      cy.delete_organization(orgId);
+    });
+
+    it('Datasets have an rdf endpoint', () => {
+        cy.request(`/dataset/${packageId}.rdf`).should((response) => {
+            expect(response.status).to.eq(200);
+            expect(response.body).to.contain('dcat:Dataset');
+        });
+    });
```

Comment:
Good test!
Should we test that the link to the RDF is in the HTML?

Reply:
I don't see that there are any links to the RDF document in our dataset pages. Were you expecting there to be one?